### ISSUE TITLE
Cloned Project not working correctly on unity 2018.3.11

### ISSUE THREAD

**Issue by Agent215** [AustinsOffice/Floating-Damage-Text-in-Unity#2]
I cloned the project successfully then run the demo scene. I am getting Two errors the first is an array out of bounds error that gets triggered on line 13 of FloatingText.cs below is the line : 
 Destroy(gameObject, clipInfo[0].clip.length);

The second error which you fixed in your video , but was not fixed in the GitHub was the text was not updating correctly but this was fixed by using the getComponent<Text>().text = text.  

So to summarize when cloning this project and running the animation does not play and the text does not update. i fixed the text update doing what you did in the video. but the animation is not playing, and yes the animator is Active!

**Comment by Agent215** [comment]
i fixed it by recreating the animation, i think it just did not get brought in to the project for some reason
